stories/4-components/chart.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+
+storiesOf('Components|Chart', module)
+  .add(
+    'Simple Counter/Default',
+    () => (
+      <>
+        <div className="counter-default counter-default--large">12</div>
+        <div className="counter-default counter-default--normal">8</div>
+        <div className="counter-default counter-default--small">2</div>
+      </>
+    ),
+    { layout: { rows: 1, cols: 3, gap: '50px' } },
+  )
+  .add(
```

Comment:
Prashant just merged a PR that includes a major update to Storybook. This has changed the Story API so writing stories looks different now. Can you rebase from master and rewrite your story using the new API syntax?

Reply:
@websmith - Sure! will update based on the latest changes